pgsql on smartproxy can grow significantly (#2546)
theforeman/foreman-documentation: guides/common/modules/ref_capsule-storage-requirements.adoc

|Directory |Installation Size |Runtime Size
ifdef::katello,satellite,orcharhino[]
|/var/lib/pulp |1 MB |300 GB
|{postgresql-lib-dir} |100 MB |20 GB
endif::[]
|/usr |3 GB |Not Applicable
|/opt/puppetlabs |500 MB |Not Applicable
|====

> The size of the `postgres` database on {SmartProxy} can grow significantly with an increasing number of lifecycle environments, content views, or repositories that the {SmartProxy} is synchronizing.

Reviewer: ```suggestion
The size of the PostgreSQL database on your {SmartProxyServer} can grow significantly with an increasing number of lifecycle environments, content views, or repositories that are synchronized from your {ProjectServer}.
```

I'd use "SmartProxyServer" based on the next sentence where you compare ProjectServer with SmartProxyServer.

Unsure if "content views" or "Content Views".

> In the largest {Project} environments, the size of `{postgresql-lib-dir}` on {SmartProxy} can grow to double or triple the size of `{postgresql-lib-dir}` on the {Project} server.

Reviewer: ```suggestion
In the largest {Project} environments, the size of `{postgresql-lib-dir}` on {SmartProxyServer} can grow to double or triple the size of `{postgresql-lib-dir}` on your {ProjectServer}.
```

|/usr |3 GB |Not Applicable
|/opt/puppetlabs |500 MB |Not Applicable
|====

> The size of the `postgres` database on {SmartProxy} can grow significantly with increasing number of lifecycles, content views, or repositories that the {SmartProxy} is synchronizing. As a consequence, the size of `/var/lib/pgsql` can double or triple.

Reviewer 2: ```suggestion
The size of the `postgres` database on {SmartProxy} can grow significantly with an increasing number of lifecycle environemnts, content views, or repositories that the {SmartProxy} is synchronizing. As a consequence, the size of `/var/lib/pgsql` can become double or triple the size of `/var/lib/pgsql` on the Foreman server.
```
